game-core/src/main/java/games/strategy/triplea/ui/menubar/help/HelpMenu.java

```diff
@@ -0,0 +1,53 @@
+package games.strategy.triplea.ui.menubar.help;
+
+import games.strategy.engine.data.GameData;
+import games.strategy.triplea.UrlConstants;
+import games.strategy.triplea.ui.UiContext;
+import java.awt.Toolkit;
+import java.awt.event.KeyEvent;
+import javax.swing.Action;
+import javax.swing.JMenu;
+import javax.swing.KeyStroke;
+import lombok.experimental.UtilityClass;
+import org.triplea.swing.SwingAction;
+import org.triplea.swing.SwingComponents;
+
+@UtilityClass
+public final class HelpMenu extends JMenu {
```

Comment:
Codacy found an issue: [Classes implementing Serializable should set a serialVersionUID](https://app.codacy.com/gh/triplea-game/triplea/pullRequest?prid=5909535)